src/hooks/useResourceYamlSchema.ts

```diff
@@ -30,7 +34,7 @@ function useResourceYamlSchema(yaml: any, resource: K8sResource | undefined) {
         enableSchemaRequest: true,
         hover: true,
         completion: true,
-        isKubernetes: true,
+        isKubernetes: resource,
```

Comment:
should this be `Boolean(resource)` ? but it's maybe redundant

Reply:
I just saw that `| undefined` was removed from the `resource` argument, so wouldn't this be always true?